src/main/java/net/sf/mzmine/modules/rawdatamethods/rawdataimport/fileformats/ZipReadTask.java

```diff
@@ -189,6 +192,8 @@ public String getTaskDescription() {
     public void cancel() {
         if (copy != null)
             copy.cancel();
+        //Is it possible to delete the temporary files here?  
+        //Otherwise canceling the task potential leaves an incompletely decompressed file
```

Comment:
I will add that, although the files should still be deleted when the JVM exits, because they were marked by 
```
tmpFile.deleteOnExit();
tmpDir.deleteOnExit();
```
See https://docs.oracle.com/javase/7/docs/api/java/io/File.html#deleteOnExit()